iface/netstack/env.go

```diff
@@ -0,0 +1,25 @@
+package netstack
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+)
+
+// IsEnabled todo: move these function to cmd layer
+func IsEnabled() bool {
+	return os.Getenv("NB_USE_NETSTACK_MODE") == "true"
+}
+
+func ListenAddr() string {
+	sPort := os.Getenv("NB_SOCKS5_LISTENER_PORT")
+	port, err := strconv.Atoi(sPort)
+	if err != nil {
```

Comment:
Let's add a warning log message with the cause to fallback to the default address. for both IFs here.